org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/editor/WEditorPopupMenu.java

```diff
@@ -62,6 +62,8 @@
     public static final String SHOWLOCATION_EVENT = "SHOW_LOCATION";
     public static final String CHANGE_LOG_EVENT = "CHANGE_LOG";
     public static final String EDITOR_EVENT = "EDITOR";
+    public static final String RESET_EVENT = "RESET";
+    public static final String COLOR_PICKER_EVENT = "COLOR_PICKER";
```

Comment:
COLOR_PICKER_EVENT constant should be moved to the WColorEditor class

Reply:
done